factory/reader.js

```diff
@@ -22,8 +22,43 @@ function parseContent(html) {
   var $ = cheerio.load(html),
       articleMarkup = $('.post-outer div.post.hentry').html(),
       content = getTextFromHtml(articleMarkup);
-
-  return content;
+      
+      //@dfasoro added this.
+      var comments = [];
+      var commentsNode = $('#comments');
+      comments.push($('H4', commentsNode).first().text());
+      
+      var authors = [], body = [], time = [];
```

Comment:
Please declared these variables on line 27 with the comments like so:
``js
var comments = [], authors = [], body = [], time = [];